ddtrace/internal/core.py

```diff
@@ -20,6 +20,102 @@
 except ImportError:
     import ddtrace.vendor.contextvars as contextvars  # type: ignore
 
+"""
+This file implements the Core API, the abstraction layer between Integrations and Product code.
+The Core API enables two primary use cases: maintaining a tree of ``ExecutionContext`` objects
+and dispatching events.
+
+When using the Core API, keep concerns separate between Products and Integrations. Integrations
```

Comment:
```suggestion
Using the Core API, keeps concerns separate between Products and Integrations. Integrations
```

? the sentence reads a tad off to me, trying to figure it out if you meant something else?

Reply:
It's an imperative, like "when you're using the api, you should keep concerns..."

Reply:
oh, reading again now makes sense, not sure why my brain wasn't working.